content/nextflow-on-aws-batch/30_batch/10_wizard.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Skip Wizard"
+chapter: false
+weight: 10
+---
+
+## Skip Wizard
```

Comment:
is this supposed to be the headline? kind of confusing. how about Open the AWS Batch console? I understand it makes sense since it's a sub-step of "Setup AWS Batch" but as a page it looks weird

Reply:
merged into the next one